TI_Practice1/Circle.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Practice1
+{
+    class Circle
```

Comment:
could you please specify the access modifier for the class? it is better to set it explicitly

Reply:
public access modifier has added